README.md

```diff
@@ -1,6 +1,8 @@
 # Prior Localization
 The repository provides the code associated with the manuscript 
-[*Brain-wide representations of prior information in mouse decision-making*](https://doi.org/10.1101/2023.07.04.547684) (Findling, Hubert et al, 2023).
+[*Brain-wide representations of prior information in mouse decision-making*](https://doi.org/10.1101/2023.07.04.547684) (Findling, Hubert et al, 2023)
+as well as [*A brain-wide map of neural activity during complex behavior*](https://doi.org/10.1101/2023.07.04.547681)
+ (International Brain Lab et al 2023).
 
 ## Dependencies
 The code has been tested on Ubuntu 20.04 and 22.04, Rocky Linux 8.8 and OSX 13.4.1, using Python 3.8, 3.9 and 3.10.
```

Comment:
We now support only 3.10, but no need to updat this as I have a few commits handling the installation on my branch. So just FYI.